### ISSUE TITLE
Example of  loading an external json file with API?

### ISSUE THREAD

**Issue by macsupport** [Kris-B/nanoGALLERY#85]
With loading a large number of images using the API method, it would be helpful to be able to load the data from an external json file. Is there any documentation  or example for this?

**Comment by Kris-B** [comment]
Not supported at this time.
Feature will be documented when implemented.

**Comment by Kris-B** [comment]
Actually, did you had a look at nanoPhotosProvider?
https://github.com/Kris-B/nanoPhotosProvider

**Comment by macsupport** [comment]
Yes, but I was interested in loading my own json file to use with a handlebars template.

**Comment by gregorybellencontre** [comment]
I'm testing nanoGallery and I have a large amount of images to handle.
I use the API way to get my albums and photos.
But it's quite long and I was wondering if a solution exists to load albums images, only when an album is opened ?
Thank by advance
